docker/grafana/provisioning/dashboards/TextileFC.json

```diff
@@ -184,9 +184,9 @@
       "tableColumn": "",
```

Comment:
All _docker/grafana/*_ changes are due to monitoring should now use docker container names instead of _localhost_. (since not running in _network-mode=host_)

Reply:
Plus, some name-changes.